python/cudf/cudf/core/series.py

```diff
@@ -1490,12 +1491,21 @@ def __neg__(self):
     @copy_docstring(CategoricalAccessor.__init__)
     @property
     def cat(self):
-        return self._column.cat(parent=self)
+        if not is_categorical_dtype(self.dtype):
+            raise AttributeError(
+                "Can only use .cat accessor " "with a 'category' dtype"
+            )
+        return CategoricalAccessor(column=self._column, parent=self)
 
     @copy_docstring(StringMethods.__init__)
     @property
     def str(self):
-        return self._column.str(parent=self)
+        return StringMethods(column=self._column, parent=self)
+
+    @copy_docstring(ListMethods.__init__)
+    @property
+    def list(self):
+        return ListMethods(column=self._column, parent=self)
```

Comment:
Should similarly validate these accessors as well

Reply:
Done